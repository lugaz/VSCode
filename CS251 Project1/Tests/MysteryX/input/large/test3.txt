getRear
deleteRear
deleteFront
insertRear 776
getFront
getRear
insertRear 70
getFront
insertFront 61
deleteRear
getFront
deleteRear
deleteRear
deleteRear
deleteFront
getRear
insertRear 23
insertFront 68
deleteRear
getFront
insertRear 942
insertFront 824
getFront
insertFront 811
getRear
getRear
deleteRear
insertFront 783
getRear
deleteFront
getFront
deleteRear
insertRear 558
deleteRear
getFront
deleteFront
getFront
insertRear 276
getFront
deleteRear
insertRear 643
insertFront 430
insertFront 32
insertFront 428
insertFront 937
deleteFront
getFront
getRear
getFront
insertFront 964
getRear
deleteFront
getFront
getFront
getFront
deleteFront
getFront
insertRear 633
deleteRear
deleteRear
insertFront 125
deleteFront
insertFront 256
getRear
getFront
getRear
deleteFront
getRear
insertRear 121
deleteFront
deleteRear
getRear
getRear
getFront
insertRear 652
getRear
getFront
insertRear 948
deleteRear
deleteRear
insertRear 905
getFront
getFront
insertFront 645
insertRear 878
deleteRear
getFront
deleteFront
deleteFront
insertRear 615
insertFront 25
deleteRear
deleteRear
insertFront 393
getRear
getFront
insertRear 456
getFront
insertFront 298
deleteRear
insertFront 676
deleteFront
deleteRear
getFront
deleteFront
insertFront 411
deleteFront
deleteFront
getFront
insertRear 535
deleteRear
insertRear 101
deleteRear
getFront
insertFront 304
deleteFront
getFront
getRear
deleteFront
getRear
insertRear 17
deleteRear
getRear
getFront
insertFront 479
getFront
getRear
insertFront 698
getFront
insertFront 858
insertRear 689
insertRear 523
deleteRear
insertFront 19
deleteFront
deleteRear
deleteRear
insertFront 44
insertFront 375
getFront
getFront
deleteFront
insertFront 729
getFront
insertFront 86
deleteRear
getRear
deleteRear
getRear
deleteFront
insertFront 864
insertFront 889
getFront
getFront
getRear
insertRear 249
getFront
deleteFront
insertFront 477
insertRear 236
getFront
getFront
getRear
getFront
getFront
insertFront 233
getFront
getRear
deleteFront
insertRear 295
deleteRear
deleteFront
deleteFront
insertFront 767
insertRear 767
getRear
insertRear 375
insertFront 366
insertFront 156
getRear
deleteFront
getRear
deleteFront
deleteRear
getFront
deleteFront
deleteRear
insertFront 174
getFront
deleteRear
insertFront 25
insertFront 153
insertRear 971
deleteRear
deleteRear
insertRear 219
insertFront 368
deleteRear
getRear
deleteFront
getFront
getFront
insertFront 966
getFront
insertFront 260
getRear
insertFront 321
getFront
getRear
getFront
insertRear 949
insertFront 907
insertFront 192
insertFront 201
insertFront 42
deleteFront
insertRear 828
getRear
deleteFront
deleteFront
insertFront 88
insertFront 2
deleteFront
getFront
deleteFront
deleteFront
insertRear 130
insertRear 291
deleteRear
getFront
getRear
deleteFront
insertRear 262
deleteRear
deleteRear
deleteFront
getRear
getFront
getRear
getFront
deleteFront
insertFront 521
deleteFront
deleteRear
getRear
deleteFront
insertFront 214
getRear
getRear
deleteFront
deleteRear
insertFront 444
deleteRear
insertRear 100
insertFront 576
insertFront 40
insertFront 858
insertRear 23
deleteFront
