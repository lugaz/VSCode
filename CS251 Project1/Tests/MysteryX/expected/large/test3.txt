-1
-1
-1
776
776
776
70
61
776
61
-1
-1
-1
23
68
824
942
942
942
68
783
811
68
558
811
811
824
824
276
937
428
643
428
643
964
428
428
428
428
32
633
643
125
824
256
824
256
824
32
121
824
824
430
652
430
948
652
430
430
878
645
645
430
615
905
824
393
393
456
676
824
298
298
411
393
25
535
101
25
304
25
25
25
-1
17
-1
-1
479
479
698
523
19
689
479
375
375
375
729
698
858
858
44
86
889
889
44
889
889
477
477
236
477
477
233
236
233
295
477
864
767
375
156
375
366
375
767
767
767
174
236
971
249
219
44
368
153
153
966
44
321
44
321
42
828
201
192
2
88
88
907
291
321
130
321
262
130
260
828
966
828
966
966
521
828
949
153
949
949
214
949
44
858
